mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1711,6 +1744,8 @@
     <string name="saved_logins_sort_strategy_last_used">Last used</string>
     <!-- Content description (not visible, for screen readers etc.): Sort saved logins dropdown menu chevron icon -->
     <string name="saved_logins_menu_dropdown_chevron_icon_content_description">Sort logins menu</string>
+    <!-- Content description (not visible, for screen readers etc.) -->
+    <string name="saved_logins_menu_dropdown_chevron_icon_content_description_2" tools:ignore="UnusedResources">Sort passwords menu</string>
```

Comment:
What is the meaning of this string? "Menu to sort passwords"? What's the element they're associated to?

Reply:
@flodolo - There is a carrot ( a 'v') next to the page header that exposes a menu to sort passwords either by name (A-Z) or last used.

Reply:
@sarah541, FYI